mvrx/src/main/kotlin/com/airbnb/mvrx/BaseMvRxViewModel.kt

```diff
@@ -28,10 +28,14 @@ abstract class BaseMvRxViewModel<S : MvRxState>(
     private val disposables = CompositeDisposable()
     private val backgroundScheduler = Schedulers.single()
     private val stateStore: MvRxStateStore<S> = MvRxStateStore(initialState)
+    private lateinit var mutableStateChecker: MutableStateChecker<S>
```

Comment:
is lateinit necessary? can it simply be `private val mutableStateChecker = MutableStateChecker(initialState)`

Reply:
you're right, it's not necessary, it was done this way to avoid calculating the hashcode  of the state object on class init in production.

that's a bit minor, but I didn't want to affect prod at all. I could add a comment about the purpose